extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -198,7 +198,7 @@ public function save() {
 		$this->response->setVal( 'status', $status );
 
 		if ( $status ) {
-			wfRunHooks( 'CuratedContentSave' );
+			wfRunHooks( 'CuratedContentSave', [ $sections ] );
```

Comment:
Will [this hook](https://github.com/Wikia/app/blob/CONCF-677/extensions/wikia/CuratedContent/CuratedContentController.class.php#L505) be ok with this change?